npm_and_yarn/lib/dependabot/npm_and_yarn/update_checker.rb

```diff
@@ -178,7 +172,19 @@ def conflicting_updated_dependencies
           )
         end
 
-        updated_deps
+        # Target dependency should be first in the result to support rebases
+        updated_deps.select { |dep| dep.name == dependency.name } +
+          updated_deps.reject { |dep| dep.name == dependency.name }
```

Comment:
This could be simplified to

```ruby
updated_deps.partition {|d| d.name == dependency.name }.inject(:+)
```
but that's probably too clever.

Reply:
Neat! I did have to google to figure that one out so I'm leaning towards sticking with this as a more beginner friendly option.